discovery-provider/src/tasks/cache_user_balance.py

```diff
@@ -200,6 +211,22 @@ def refresh_user_ids(
                         associated_balance += (
                             balance + delegation_balance + stake_balance
                         )
+                    for wallet in wallets["associated_wallets"]["sol"]:
+                        root_sol_account = PublicKey(wallet)
+                        derived_account, _ = PublicKey.find_program_address(
+                            [
+                                bytes(root_sol_account),
+                                bytes(SPL_TOKEN_ID_PK),
+                                bytes(WAUDIO_PROGRAM_PUBKEY),
```

Comment:
since this is subject to change with wormhole v2, just want to confirm that we won't' _actually_ be enabling linking right? cc @raymondjacobson

Reply:
yeah -- this won't be exposed in the UI for a bit (timed with NFT stuff probably and this change here will still work for NFT support, just not wAudio balance). we need to be watching v2 very closely and update this env var as soon as we can.

Reply:
good callout. we should be good though